typings/vscode-mssql.d.ts

```diff
@@ -1078,6 +1079,14 @@ declare module 'vscode-mssql' {
 		defaultDeploymentOptions: DeploymentOptions;
 	}
 
+	export interface SavePublishProfileParams {
+		profilePath: string;
+		databaseName: string;
+		connectionString: string;
+		sqlCommandVariableValues?: Record<string, string>;
```

Comment:
@SakshiS-harma should this be a `Map` after the conversion PR from earlier this week?

Reply:
I just updated it in the latest commit. Lmk if it shouldn't be swapped over to Map @SakshiS-harma